packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -227,13 +276,19 @@ const { requiredMarkerClass, requiredTypeClass } = useRequired(props);
     }
   }
 
-  &__loading {
-    color: var(--onyx-color-text-icons-primary-intense);
-  }
-
-  &__footer {
-    width: 100%;
-    color: var(--onyx-color-text-icons-neutral-soft);
+  &-skeleton {
+    display: flex;
+    flex-direction: column;
+    gap: var(--onyx-spacing-5xs);
+    &__label {
+      width: var(--onyx-spacing-3xl);
+      height: 1.25rem;
+    }
+    &__input {
+      width: 17rem;
```

Comment:
```suggestion
      width: 16rem;
```

Reply:
17 is such a uneven number :( We should sticky to multiples of 2 or 4

Reply:
it's in the design